### ISSUE TITLE
Couldn't find preset "es2015" relative to directory

### ISSUE THREAD

**Issue by track0x1** [developit/preact-render-to-string#56]
I have my own `.babelrc` config (which doesn't include `babel-preset-es2015`)... why is preact requiring me to install and use the es2015 preset specifically?

```
ERROR in ./node_modules/preact-render-to-string/dist/index.js
    Module build failed: Error: Couldn't find preset "es2015" relative to directory "/Users/me/dev/app/node_modules/preact-render-to-string"
```

**Comment by studentIvan** [comment]
Same error

@track0x1 `npm i --save babel-preset-es2015 babel-preset-stage-0` fixed this problem, but these presets are deprecated

**Comment by developit** [comment]
This should be fixed in 3.8.
